api/prometheus/api.go

```diff
@@ -280,8 +280,10 @@ func (qr *queryResult) UnmarshalJSON(b []byte) error {
 type QueryAPI interface {
 	// Query performs a query for the given time.
 	Query(ctx context.Context, query string, ts time.Time) (model.Value, error)
-	// Query performs a query for the given range.
+	// QueryRange performs a query for the given range.
 	QueryRange(ctx context.Context, query string, r Range) (model.Value, error)
+	// QueryLabelValues performs a query for the values of the given label.
+	QueryLabelValues(ctx context.Context, label string) ([]string, error)
```

Comment:
To be more consistent with naming we have elsewhere, maybe omit the `Query` and just call this `LabelValues`.